Issue #215 - Attestation Results (#273)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

The quality of the Attestation Results depend upon the ability of the Verifier to evaluate the Attester.
Different Attesters have a different _Strength of Function_ {{strengthoffunction}}, which results in the Attestation Results being qualitatively different in strength.

An Attestation Result that indicates non-compliance can be used by an Attester (in the passport model) or
a Relying Party (in the background-check model) to indicate that the Attester
should not be treated as authorized and may be in need of remediation.  In some cases,
it may even indicate that the Evidence itself cannot be authenticated as being correct.

> By default, the Relying Party assumes the Attester does not comply with Verifier appraisals. 

Reviewer: This seems really awkward.

```suggestion
By default, the Relying Party does not believe the Attester to be compliant. 
```

Reviewer 2: +1 to mcr's suggested text

> Upon receipt of an authentic Attestation Result and given the Appraisal Policy for Relying Party is satisfied. 
> The Attester is allowed to perform the prescribed actions or access. 

Reviewer: ```suggestion
Upon receipt of an authentic Attestation Result and given the Appraisal Policy for Relying Party is satisfied, then
the Attester is allowed to perform the prescribed actions or access. 
```

Reviewer 2: Better, but "Appraisal Policy for Relying Party" also needs to be "Appraisal Policy for Attestation Results" which is the defined term.

Reviewer 3: One more question: Checking the Attestation Result's authenticity is included in the Appraisal Policy for Attestation Results or not?
I.e., should we call an Attestation Result authentic only after the Appraisal Policy for Attestation Results is satisfied?

Reviewer: okay, what wording change do you suggest?
